src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -339,8 +339,8 @@ private void ProcessExpressionParameter()
             if (ExcludeProperty != null)
```

Comment:
And above we use `Property.Length != 0` Maybe better `ExcludeProperty.Length != 0` ?